### ISSUE TITLE
On zooming less than min zoom the image is panned to center

### ISSUE THREAD

**Issue by jitheeshsk** [jayrambhia/CropperNoCropper#21]
Set min zoom to 0.8f. 
Take an image which is really long in height, i'm using the below image.

Issue seems to be in on up
 private boolean onUp(){
...
            if (showAnimation()) {

                animateAdjustmentWithScale(tx, xx, ty, **ty,** scaleX, mMinZoom);

            } else {
                matrix.reset();
                matrix.setScale(mMinZoom, mMinZoom);
                matrix.postTranslate(xx, **ty**);
                setImageMatrix(matrix);
                invalidate();
                if (DEBUG) {
                    Log.i(TAG, "scale after invalidate: " + getScale(matrix));
                }
            }
            return true;
        }
...

Should solve the issue, 

![screenshot_20170420-125705](https://cloud.githubusercontent.com/assets/6497623/25220848/b53187d0-25d0-11e7-921f-d1b410427699.png)

**Comment by jayrambhia** [comment]
Yes, that is the desired behavior. It's a cropper so it tires to pan the image to center.

**Comment by jitheeshsk** [comment]
Shouldn't the state of view be maintained, it should pan to center of screen not image right, just a suggestion

**Comment by jayrambhia** [comment]
@jitheeshsk I don't understand. How can image pan to center of itself? Can you add a gif/video? That would help me understand the problem.

**Comment by jitheeshsk** [comment]
Attached is the gif showing the issue

![image_crop](https://cloud.githubusercontent.com/assets/6497623/25223592/672fa4e6-25d9-11e7-9ffa-01863278f913.gif)

**Comment by jayrambhia** [comment]
@jitheeshsk Thanks for the gif. This is indeed a problem. Can you tell me which version of the lib are you using?

**Comment by jitheeshsk** [comment]
It is 0.1.8

**Comment by jayrambhia** [comment]
@jitheeshsk I'm unable to reproduce it in 0.1.8 with min zoom 0.8f.

![crop_out](https://cloud.githubusercontent.com/assets/1256649/25224552/061b8fea-25bf-11e7-8e47-bc36f8ed5440.gif)

**Comment by jitheeshsk** [comment]
Hi the min zoom is not set, else the image cannot be the completely zoomed
out. Check the min zoom value.

I set min zoom in cropperview.post(). As the minZoomValue is rewritten in
onLayout of cropperImageView.

**Comment by jayrambhia** [comment]
@jitheeshsk oh yes, got the problem. I'll try to fix it. And also `setMinZoom` should work out of the box.

**Comment by jitheeshsk** [comment]
@jayrambhia I will also see if there is a work around, will try to post if i am able to resolve the issue.

Thanks

**Comment by jitheeshsk** [comment]
in onUp()


```
f (scaleX < mMinZoom) {
            if (DEBUG) {
                Log.i(TAG, "set scale: " + mMinZoom);
            }
float yy = ty * mMinZoom/scaleX;
            if (showAnimation()) {
                if(yy > 0)
                    yy = 0;
                animateAdjustmentWithScale(tx, xx, ty, yy, scaleX, mMinZoom);

            }
}
```


Fixed the issue for me

**Comment by jayrambhia** [comment]
Fixed in 0.1.9.

@jitheeshsk
